src/vendor/jitsi/index.js

```diff
@@ -0,0 +1,2 @@
ant-joined":this._participants[r]=this._participants[r]||{},this._participants[r].displayName=n.displayName,this._participants[r].formattedDisplayName=n.formattedDisplayName,D(this,1);break;case"participant-left":D(this,-1),delete this._participants[r];break;case"display-name-change":{const e=this._participants[r];e&&(e.displayName=n.displayname,e.formattedDisplayName=n.formattedDisplayName);break}case"email-change":{const e=this._participants[r];e&&(e.email=n.email);break}case"avatar-changed":{const e=this._participants[r];e&&(e.avatarURL=n.avatarURL);break}case"on-stage-participant-changed":this._onStageParticipant=r,this.emit("largeVideoChanged");break;case"large-video-visibility-changed":this._isLargeVideoVisible=n.isVisible,this.emit("largeVideoChanged");break;case"video-conference-left":D(this,-1),delete this._participants[this._myUserID]}const i=M[t];return!!i&&(this.emit(i,n),!0)})}addEventListener(e,t){this.on(e,t)}addEventListeners(e){for(const t in e)this.addEventListener(t,e[t])}dispose(){this.emit("_willDispose"),this._transport.dispose(),this.removeAllListeners(),this._frame&&this._frame.parentNode&&this._frame.parentNode.removeChild(this._frame)}executeCommand(e,...t){e in N?this._transport.sendEvent({data:t,name:N[e]}):console.error("Not supported command name.")}executeCommands(e){for(const t in e)this.executeCommand(t,e[t])}getAvailableDevices(){return Object(I.a)(this._transport)}getCurrentDevices(){return Object(I.b)(this._transport)}isAudioAvailable(){return this._transport.sendRequest({name:"is-audio-available"})}isDeviceChangeAvailable(e){return Object(I.c)(this._transport,e)}isDeviceListAvailable(){return Object(I.d)(this._transport)}isMultipleAudioInputSupported(){return Object(I.e)(this._transport)}invite(e){return Array.isArray(e)&&0!==e.length?this._transport.sendRequest({name:"invite",invitees:e}):Promise.reject(new TypeError("Invalid Argument"))}isAudioMuted(){return this._transport.sendRequest({name:"is-audio-muted"})}getAvatarURL(e){const{avatarURL:t}=this._participants[e]||{};return t}getDisplayName(e){const{displayName:t}=this._participants[e]||{};return t}getEmail(e){const{email:t}=this._participants[e]||{};return t}_getFormattedDisplayName(e){const{formattedDisplayName:t}=this._participants[e]||{};return t}getIFrame(){return this._frame}getNumberOfParticipants(){return this._numberOfParticipants}isVideoAvailable(){return this._transport.sendRequest({name:"is-video-available"})}isVideoMuted(){return this._transport.sendRequest({name:"is-video-muted"})}removeEventListener(e){this.removeAllListeners(e)}removeEventListeners(e){e.forEach(e=>this.removeEventListener(e))}sendProxyConnectionEvent(e){this._transport.sendEvent({data:[e],name:"proxy-connection-event"})}setAudioInputDevice(e,t){return Object(I.f)(this._transport,e,t)}setAudioOutputDevice(e,t){return Object(I.g)(this._transport,e,t)}setVideoInputDevice(e,t){return Object(I.h)(this._transport,e,t)}_getElectronPopupsConfig(){return Promise.resolve(S)}}}])}));
```

Comment:
Can we use npm to install this dependency?

Reply:
no. there is no official package. but I also added 
https://github.com/aeternity/superhero-ui/blob/8d7893d39796851c887ce948658c6c826bbe2fba/package.json#L16

so we can use like
https://github.com/aeternity/superhero-ui/blob/8d7893d39796851c887ce948658c6c826bbe2fba/src/views/Conference.vue#L12

and also change webpack config a little to support the aliases

Reply:
@davidyuk not sure how this will works in Windows for exmaple
https://github.com/aeternity/superhero-ui/blob/8d7893d39796851c887ce948658c6c826bbe2fba/package.json#L16

hope it will.. 

I found some packages when I work on that, but there so no official and some of them are without repo and obfuscate, so I don't trust them

Reply:
ok, where can I find sources of the package that is used here?

Reply:
@davidyuk https://github.com/jitsi/jitsi-meet/blob/master/modules/API/external/external_api.js